go-controller/pkg/ovn/hybrid.go

```diff
@@ -58,7 +58,7 @@ func (oc *DefaultNetworkController) handleHybridOverlayPort(node *kapi.Node, ann
 		// No subnet allocated yet; clean up
 		klog.V(5).Infof("No subnet allocation yet for %s", node.Name)
 		if annotationOK {
-			oc.deleteHybridOverlayPort(node)
+			_ = oc.deleteHybridOverlayPort(node)
```

Comment:
dont we want to return error here since there was some problem deleting things and it will retry?

Reply:
done